src/bin/driver.rs

```diff
@@ -201,7 +182,7 @@ mod app {
         let clock = SystemTimer::new(|| monotonics::now().ticks() as u32);
 
         // Configure the microcontroller
-        let (stabilizer, _pounder) = hardware::setup::setup(
+        let (stabilizer, _pounder, driver) = hardware::setup::setup(
```

Comment:
Maybe `hardware::setup::setup()` should return (stabilizer, mezzanine), where `mezzanine` is the driver for the detected mezzanine card?

Reply:
Right. A `enum { None, Pounder(Pounder), Driver(Driver) }`